docs/ramalama-run.1.md

```
@@ -0,0 +1,581 @@
+
+[//]: # (BEGIN included file options/model-transports.md)
+## MODEL TRANSPORTS
+
+| Transports    | Prefix | Web Site                                            |
+| ------------- | ------ | --------------------------------------------------- |
+| URL based     | https://, http://, file:// | `https://web.site/ai.model`, `file:///tmp/ai.model`|
+| HuggingFace   | huggingface://, hf://, hf.co/ | [`huggingface.co`](https://www.huggingface.co)|
+| ModelScope    | modelscope://, ms:// | [`modelscope.cn`](https://modelscope.cn/)|
+| Ollama        | ollama:// | [`ollama.com`](https://www.ollama.com)|
+| rlcr          | rlcr://   | [`ramalama.com`](https://registry.ramalama.com) |
+| OCI Container Registries | oci://, docker:// | [`opencontainers.org`](https://opencontainers.org)||||Examples: [`quay.io`](https://quay.io),  [`Docker Hub`](https://docker.io),[`Artifactory`](https://artifactory.com)|
```

markdownlint-cli2 (0.23.1)

[warning] Table column count
Expected: 3; Actual: 7; Too many cells, extra data will be missing

(MD056, table-column-count)